app/src/main/java/io/homeassistant/companion/android/sensors/NextAlarmManager.kt

```diff
@@ -67,6 +66,8 @@ class NextAlarmManager : SensorManager {
                 val sdf = SimpleDateFormat(dateFormat)
                 sdf.timeZone = TimeZone.getTimeZone("UTC")
                 utc = sdf.format(Date(triggerTime))
+
+                pendingIntent = alarmClockInfo.showIntent.creatorPackage.toString()

```

Comment:
Lets make it null safe rather than let it throw an exception every time.
```suggestion
                pendingIntent = alarmClockInfo.showIntent?.creatorPackage?: "Unknown"

```